service/controller/resource/tccpn/create.go

```diff
@@ -611,3 +617,11 @@ func idFromSubnets(subnets []*ec2.Subnet, name string) string {
 
 	return ""
 }
+
+func stackComplete(status string) bool {
```

Comment:
Added helper function to check for the status

Reply:
We could add these functions in the key or cloud formation package so that we reuse the code where possible.

Reply:
Makes totally sense, I moved it to key since we already use it and it doesn't collide with cloudformation package (external vs internal)